### PR TITLE
:hammer: (#819) Invite users when creating workspace

### PR DESCRIPTION
**Pull request by hoangdat**
https://user-images.githubusercontent.com/6462404/142168546-f8521d69-b77b-4932-9d20-263d8251821c.mp4

### CODE REVIEW

File: lib/blocs/workspaces_cubit/workspaces_cubit.dart [hunk 1]
```diff
@@ -76,6 +79,17 @@ class WorkspacesCubit extends Cubit<WorkspacesState> {
     final workspace = await _repository.create(
         companyId: companyId, name: name, members: members);
 
+    if (members != null) {
```
[the remark this answers is not in the document]
Comment (hoangdat) [reply lib/blocs/workspaces_cubit/workspaces_cubit.dart:82]: we don't need to do that. Please run the below test
```
  List<int> mapEmptyList(List<String>? members) {
    if (members != null) {
      return members.map((member) => member.length).toList();
    }
    throw UnimplementedError('error');
  }
  
  test('empty list map', () {
    assert(mapEmptyList(['user1', 'user2', 'user3']).length == 3);

    assert(mapEmptyList(List<String>.empty()).length == 0);
  });
```

[the remark this answers is not in the document]
Comment (huynguyennovem) [reply lib/blocs/workspaces_cubit/workspaces_cubit.dart:82]: I mean that it doesn't need to await a task for an empty list. Once you passing empty list, you have to wait it completed even though it does nothing (no member will be added)

File: lib/blocs/magic_link_cubit/invitation_email_cubit/invitation_email_cubit.dart [hunk 2]
```diff
@@ -49,7 +50,7 @@ class InvitationEmailCubit extends Cubit<InvitationEmailState> {
         // Sent email with API
         List<EmailInvitationResponse> resultList = [];
         try {
-          resultList = await _workspacesRepository.inviteUser(invitationList);
+          resultList = await _workspacesRepository.inviteUser(Globals.instance.companyId!, Globals.instance.workspaceId!, invitationList);
```
[the remark this answers is not in the document]
Comment (hoangdat) [reply lib/blocs/magic_link_cubit/invitation_email_cubit/invitation_email_cubit.dart:53]: I parameterize for `inviteUser` method. we can use it in many cases

File: lib/blocs/workspaces_cubit/workspaces_cubit.dart (hunk 1 above)
[the remark this answers is not in the document]
Comment (hoangdat) [reply lib/blocs/workspaces_cubit/workspaces_cubit.dart]: update :)

**Review by huynguyennovem — APPROVED** [review]
/lgtm

**Pull request merged**